src/IconButton/_IconButton.scss

```diff
@@ -52,12 +56,3 @@ $icon-color-invert: white !default;
         }
     }
 }
-
-.iconbutton-refwrapper {
-    height: $icon-button-refwrapper-size;
-    width: $icon-button-refwrapper-size;
-    padding: $icon-button-margin * 0.5;
-    button {
-        border: none;
-    }
-}
```

Comment:
Removing this rewrapper until we find we need it again. It was kind of in the way of passing through custom classNames on the icon.